hieradata_aws/common.yaml

```diff
@@ -782,7 +782,7 @@ govuk::apps::transition::redis_port: "%{hiera('sidekiq_port')}"
 govuk::apps::travel_advice_publisher::redis_host: "%{hiera('sidekiq_host')}"
 govuk::apps::travel_advice_publisher::redis_port: "%{hiera('sidekiq_port')}"
 
-govuk::apps::account_api::db_hostname: "postgresql-primary"
+govuk::apps::account_api::db_hostname: "account-api-postgresql-primary"
```

Comment:
I don't think we should change this straight away - as we'd want the postgresql-primary to still be the db in use until we're happy to switchover.